tools/src/main/java/org/apache/kafka/tools/DelegationTokenCommand.java

```diff
@@ -202,14 +206,20 @@ static class DelegationTokenCommandOptions extends CommandDefaultOptions {
         public DelegationTokenCommandOptions(String[] args) {
             super(args);
 
-            String bootstrapServerDoc = "REQUIRED: server(s) to use for bootstrapping.";
+            String bootstrapServerDoc = "REQUIRED: server(s) to use for bootstrapping. When the --bootstrap-controller argument is used --bootstrap-servers must not be specified.";
             String commandConfigDoc = "REQUIRED: A property file containing configs to be passed to Admin Client. Token management" +
                     " operations are allowed in secure mode only. This config file is used to pass security related configs.";
 
             this.bootstrapServerOpt = parser.accepts("bootstrap-server", bootstrapServerDoc)
                     .withRequiredArg()
                     .ofType(String.class);
 
+            this.bootstrapControllerOpt = parser.accepts("bootstrap-controller",
+                                                         "REQUIRED: A comma-separated list of bootstrap.controllers that can be supplied instead of bootstrap-servers."
+                                                         + " This is useful for administrators who wish to bypass the brokers.")
+                                            .withRequiredArg()
```

Comment:
You cannot specify both `--bootstrap-server` and `--bootstrap-controller` as required. This makes result like:

```
>  ./bin/kafka-delegation-tokens.sh --bootstrap-controller localhost:9093 --describe
Missing required argument "[bootstrap-server]"
```

Please using `withOptionalArg` and do the check in `checkArgs`.